integration_testing/features/super-admin/super-admin-setting-port.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Super admin set the kolibri server port
+    Super admin should be able to set the koliri server port into different ports
+
+  Background:
+    Given That I have or don't have an options.ini file with an alternate port set
+      And I have or don't have the KOLIBRI_HTTP_PORT environment variable set
```

Comment:
This is confusing, can you re-phrase these "I have or don't have" statements?  Since they look optional to me, then there's no need for these to become `Given`s.

Maybe one `Given` is that user is logged-in as super admin?

Reply:
> Maybe one Given is that user is logged-in as super admin?

I think it's irrelevant for the super admin to  logged-in as given. The super admin don't need to log in and change the Kolibri server port.